woocommerce/woocommerce
# Remove phpcs Github action in favor of running phpcs directly

### Changes proposed in this Pull Request:

Some users have been having trouble with the PHPCS action failing in their PRs. When I investigated I found the way we rely on an action is quite limiting. We can't do things like increase timeout or set arbitrary settings on PHPCS. So i think it would be good to run PHPCS directly and use another action to just pass in the list of files changed.

I've also made it so the lint doesn't do any setup work if the file changes aren't there, which should save a lot of time.



### How to test the changes in this Pull Request:

You can see the CI pass since I've included some PHP updates from another PR here to confirm it works. But to do further testing you can:

1. Fork the repo
2. In your fork enable actions
3. Create a new branch with changes from this branch
4. Add some other PHP changes you'd like to see linted
5. PR your new branch into your fork
6. Monitor lint output in the checks as they run.




### FOR PR REVIEWER ONLY:
